Letterbook.Core/PostService.cs

```diff
@@ -104,11 +104,20 @@ public async Task<Post> Draft(Post post, Uuid7? inReplyToId = default, bool publ
 			content.Sanitize(_sanitizers);
 		}
 
+		if (post.Audience.Contains(Audience.Public))
+		{
+			post.Audience = post.Audience.ReplaceWith(post.Creators.Select(Audience.Followers).ToHashSet());
```

Comment:
This seems like it replaces public posts with followers-only. Is that correct / intended?

Reply:
Actually, no, it wasn't. That is a good catch. The idea is to add the followers audience if it wasn't specified, but we still want to retain the public audience.